module/os/windows/zfs/zfs_vnops_windows.c

```diff
@@ -132,6 +132,11 @@ zfs_AcquireForLazyWrite(void *Context, BOOLEAN Wait)
 	struct vnode *vp = fo->FsContext;
 	dprintf("%s:fo %p\n", __func__, fo);
 
+	if (unlikely(zfsvfs == NULL)) {
+		dprintf("%s: fo %p already freed zfsvfs\n", __func__, fo);
+		return (FALSE);
+	}
+
```

Comment:
Looks good, but there are 3 nearly identical calls. AcquireLazyWrite, AcquireReadAhead and at the end, fastio_acquire_for_mod_write.

Reply:
Updated